airsonic-main/src/main/java/org/airsonic/player/service/metadata/JaudiotaggerParser.java

```diff
@@ -140,7 +140,7 @@ public MetaData getRawMetaData(File file) {
                                     }
                                     break;
                                 case ID3v24Frames.FRAME_ID_ARTIST:
-                                    if (null == metaData.getArtist() && null == songArtist) {
+                                    if (null == metaData.getArtist()) {
```

Comment:
Remove unnecessary judgments.

Reply:
I agree with that opinion!
However, I did this because the method of writing in this class is not uniform.
 - Rewrite the place related to this commit with StringUtils
 - Rewrite the null check of this class with StringUtils

Which one is better ...?

Reply:
Not sure I understand what option 2 is, but in general I would say that new code should use StringUtils.isBlank when it makes sense.

Reply:
A correction has been made.

https://github.com/airsonic/airsonic/blob/1c4a70af99229091c258033f9640e4bc27815ce4/airsonic-main/src/main/java/org/airsonic/player/service/metadata/JaudiotaggerParser.java#L112-L155